src/test/scala/com/netaporter/precanned/BaseSpec.scala

```diff
@@ -6,14 +6,14 @@ import akka.http.scaladsl.model._
 import akka.stream.ActorMaterializer
 
 import scala.concurrent.duration._
-import scala.concurrent.Future
+import scala.concurrent.{ ExecutionContextExecutor, Future }
 
 trait BaseSpec {
-  implicit val system = ActorSystem()
-  implicit val ec = system.dispatcher
-  implicit val materializer = ActorMaterializer()
+  implicit val system: ActorSystem = ActorSystem()
+  implicit val ec: ExecutionContextExecutor = system.dispatcher
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
```

Comment:
It should be just `Materializer` - no need to expose the concrete impl  type as an implicit type.